### PR TITLE
network: TC - introduce DRR

### PR DESCRIPTION
**Pull request by ssahani**
please see http://man7.org/linux/man-pages/man8/tc-drr.8.html

cc @yuwata

### CODE REVIEW

File: src/network/tc/drr.c [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        r = parse_size(rvalue, 1000, &u);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r,
+                           "Failed to parse '%s=', ignoring assignment: %s",
+                           lvalue, rvalue);
+                return 0;
+        }
+
+        drr->quantum = (uint32_t) u;
```
Comment (yuwata) [inline src/network/tc/drr.c:93]: Please check the range of the value before casting:
```
if (u > UINT32_MAX) {
    log_syntax(...);
    return 0;
}
```

File: man/systemd.network.xml [hunk 2]
```diff
@@ -2707,6 +2707,41 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[DRR] Section Options</title>
```
Comment (yuwata) [inline man/systemd.network.xml]: The section name should be `[DeficitRoundRobinScheduler]`, I think.

File: man/systemd.network.xml [hunk 3]
```diff
@@ -2707,6 +2707,41 @@
...
+          Takes a number in hexadecimal ranges 1 to ffff. Defaults to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Quantum=</varname></term>
+        <listitem>
+          <para>Specifies the amount of bytes a flow is allowed to dequeue before the
+          scheduler moves to the next class. An unsigned integer ranges 1 to 4294967294.
+          Defaults to the MTU of the interface.</para>
```
Comment (yuwata) [inline man/systemd.network.xml:2739]: Default is unset?

Comment (ssahani) [reply man/systemd.network.xml:2739]: No it's correct.

**Comment by ssahani** [comment]
updated thanks for the review @yuwata

**Pull request closed without merging**